docs/develop/web-client-performance.md

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Improve the performance of your Office Scripts'
+description: 'How to create faster scripts through understanding how the script communicates with the Excel workbook.'
```

Comment:
This seems kinda long?

Reply:
Fair. I improved it, though it's not really shorter. FWIW, CATS has minimum and maximum description lengths, so it'll ding this if there's a problem.